specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json

```diff
@@ -0,0 +1,4472 @@
+        "Devices",
+        "Modules",
+        "All"
+      ],
+      "x-ms-enum": {
+        "name": "ImportType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Devices",
+            "name": "Devices",
+            "description": "Import only devices but not modules."
+          },
+          {
+            "value": "Modules",
+            "name": "Modules",
+            "description": "Import only modules but not devices."
+          },
+          {
+            "value": "All",
+            "name": "All",
+            "description": "Import both devices and modules."
+          }
+        ]
+      }
+    },
+    "StepResult": {
+      "type": "object",
+      "properties": {
+        "update": {
+          "$ref": "#/definitions/UpdateInfo",
+          "description": "The update that this step installs if it is of reference type."
+        },
+        "description": {
+          "type": "string",
+          "description": "Step description."
+        },
+        "resultCode": {
+          "type": "integer",
+          "description": "Install result code."
+        },
+        "extendedResultCode": {
+          "type": "integer",
+          "description": "Install extended result code"
+        },
+        "resultDetails": {
+          "type": "string",
+          "description": "A string containing further details about the install result"
+        }
+      },
+      "required": [
+        "resultCode",
+        "extendedResultCode"
+      ],
+      "description": "The step result under an update."
+    },
+    "InstallResult": {
+      "type": "object",
+      "properties": {
+        "resultCode": {
+          "type": "integer",
+          "description": "Install result code."
+        },
+        "extendedResultCode": {
+          "type": "integer",
+          "description": "Install extended result code"
+        },
+        "resultDetails": {
+          "type": "string",
+          "description": "A string containing further details about the install result"
+        },
+        "stepResults": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StepResult",
+            "description": "Step results"
+          },
+          "description": "Array of step results"
+        }
+      },
+      "required": [
+        "resultCode",
+        "extendedResultCode"
+      ],
+      "description": "The install result of an update and any step results under it."
+    },
+    "DeviceUpdateAgentId": {
+      "type": "object",
+      "properties": {
+        "deviceId": {
+          "type": "string",
+          "description": "Device Id"
+        },
+        "moduleId": {
+          "type": "string",
+          "description": "Module Id"
+        }
+      },
+      "required": [
+        "deviceId"
+      ],
+      "description": "Device Update agent id"
+    },
+    "LogCollection": {
+      "type": "object",
+      "properties": {
+        "operationId": {
```

Comment:
Can we just change this property to "logCollectionId" since that's clearly the preferred name.

Reply:
See: https://github.com/Azure/azure-rest-api-specs/pull/20023#discussion_r939428810

Reply:
It would require writing new controllers returning this new data type. The team triaged this and decided that it doesn't meet our GA bar at this point. We will need to deal with this in a future release.